rollbar/__init__.py

```diff
@@ -864,11 +866,14 @@ def _add_locals_data(data, exc_info):
             # Fill in all of the named args
             for named_arg in named_args:
                 if named_arg in local_vars:
-                    args.append(_transform(local_vars[named_arg], key=(named_arg,)))
+                    args[named_arg] = local_vars[named_arg]
 
             # Add any varargs
             if arginfo.varargs is not None:
-                args.extend(local_vars[arginfo.varargs])
+                if SETTINGS['locals']['scrub_varargs']:
```

Comment:
*args could contain anything, including passwords so the safest thing is to scrub by default